infra.lvn_tools/extend_boot/meta/runtime.yml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
You can remove this file as all its lines are commented out

Reply:
Done